### ISSUE TITLE
Play widget is not triggering traits updates

### ISSUE THREAD

**Issue by gabrielelanaro** [ipython/ipywidgets#676]
Somehow the Play widget does not trigger updates for traits

```
from ipywidgets import Play
play = Play(max=10)
play.observe(lambda hello: print(hello))
play
```

```
import ipywidgets
import jupyter_client
print(ipywidgets.__version__)
print(jupyter_client.__version__)

5.2.2
4.3.0
```

**Comment by SylvainCorlay** [comment]
@gabrielelanaro thanks I can reproduce. It is probably a backport issue since it works in the master branch.
The following workaround should allow you to observe a linked slider in the meantime. 
```
from ipywidgets import *
play = Play(max=100)
s = IntSlider()
jslink((play, 'value'), (s, 'value'))
play
```

**Comment by SylvainCorlay** [comment]
So, oddly this works perfectly in master and this bugs seems to be ipywidgets 5.x only.

**Comment by jasongrout** [comment]
Closed as working on master, which will be released as soon as 6.0. Thanks for bringing this up!
